backend/app/api/game.py

```diff
@@ -160,7 +160,20 @@ def get_tile():
 
 def filter_message(data: Dict[str, str]) -> Dict[str, str]:
     """function to filter chat messages"""
-
+    
+    bannedWords = ["shit","fuck","crap","bitch"] # will add more words to this
+    
+    chatMessage = data.get('message')
+    censoredMessageList = []
+    
+    for word in chatMessage.split():
+        if word in bannedWords:
+            censoredMessageList.append("[redacted]")
+        else:
+         censoredMessageList.append(word)
+    
+    censoredMessage = " ".join(censoredMessageList)
+    data.update({'message': censoredMessage})
```

Comment:
You could just change this line to `data['message'] = censoredMessage`. dict.update() is normally used for combining two dictionaries together rather than just updating a single value in a dict.